spanner/api/Spanner.Samples/CopyBackup.cs

```diff
@@ -0,0 +1,55 @@
+﻿// Copyright 2022 Google Inc.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+// [START spanner_copy_backup]
+
+using Google.Api.Gax;
+using Google.Cloud.Spanner.Admin.Database.V1;
+using Google.Protobuf.WellKnownTypes;
+using System;
+
+public class CopyBackupSample
+{
+    public void CopyBackup(string sourceInstanceId, string sourceProjectId, string sourceBackupId, string targetInstanceId, string targetProjectId, string targetBackupId, DateTimeOffset expireTime, DateTimeOffset maxExpireTime)
+    {
+        DatabaseAdminClient databaseAdminClient = new DatabaseAdminClientBuilder { Endpoint = "staging-wrenchworks.sandbox.googleapis.com" }.Build();
```

Comment:
Do we still need to use the sandbox endpoint or is this feature available already?